src/components/HomePage/SearchArea/SearchArea.js

```diff
@@ -36,7 +38,8 @@ export const AdvancedSearchControllerContext = React.createContext({});
 export const AdvancedSearchController = ({
     enableAdvancedSearchDefault, showJobDurationSlider, setShowJobDurationSlider, jobMinDuration,
     jobMaxDuration, setJobDuration, jobType, setJobType, fields, setFields, technologies, setTechs,
-    resetAdvancedSearchFields, onSubmit, searchValue, setSearchValue, onMobileClose,
+    resetAdvancedSearchFields, onSubmit, searchValue, setSearchValue, onMobileClose, showHidden,
+    setShowHidden,
 }) => {
 
     const {
```

Comment:
As discussed in person, the `showHidden` field should also appear in the URL, i.e., you should change the hook to include the `showHidden` function.